mongodbatlas/data_source_mongodbatlas_network_container.go

```diff
@@ -124,6 +131,10 @@ func dataSourceMongoDBAtlasNetworkContainerRead(d *schema.ResourceData, meta int
 		return fmt.Errorf("error setting `vnet_name` for Network Container (%s): %s", d.Id(), err)
 	}
 
+	if err = d.Set("regions", container.Regions); err != nil {
```

Comment:
I mean , if it's azure, aws or gcp, the api returns always the regions field? (even empty)

Reply:
yea, I see the client code, curious do we get any warning, because moving from nil to empty type?
	Regions             []string `json:"regions,omitempty"`    // Regions are available for GCP